ports/esp32/boards/GARATRONIC_PYBSTICK26_ESP32C3/modules/pybstick26_esp32c3.py

```diff
@@ -0,0 +1,29 @@
+# PYBSTICK26-ESP32C3 MicroPython Helper Library
```

Comment:
I think it would be better to call this file `board.py` to make it a bit easier to import and use.  Eg:
```py
import board
board.led.on()
```